src/condor_scripts/CondorPersonal.pm

```diff
@@ -1240,6 +1240,7 @@ debug( "HMMMMMMMMMMM personal local is $personal_local , mytoppath is $mytoppath
 				print NEW "JOB_START_DELAY = 0\n";
 				print NEW "DAGMAN_USER_LOG_SCAN_INTERVAL = 1\n";
 				print NEW "LOCK = \$(LOG)\n";
+				print NEW "FILETRANSFER_PLUGINS = \$(LIBEXEC)/curl_plugin\n";
```

Comment:
not here, this one should be in the nmi_tools/glue/test/remote_pre.pl at about  line 200